indico/modules/rb/templates/emails/reservations/creation_email_to_user.txt

```diff
@@ -1,31 +1,76 @@
 {% extends 'rb/emails/reservations/base_email_to_user.txt' %}
 {% set excluded_days = reservation.find_excluded_days().count() %}
+{%- set prebooking = not reservation.is_accepted -%}
 
-{% block body -%}
+{% block subject -%}
+    {% if prebooking -%}
+        {% trans room=reservation.room.full_name, start=reservation.start_dt|format_datetime('dd/MM/yyyy HH:mm', locale='en_GB') -%}
+            [{{ room }}] Pre-Booking awaiting acceptance ({{ start }})
+        {%- endtrans %}
+    {%- else -%}
+        {% trans room=reservation.room.full_name, start=reservation.start_dt|format_datetime('dd/MM/yyyy HH:mm', locale='en_GB') -%}
+            [{{ room }}] Booking confirmed ({{ start }})
+        {%- endtrans %}
+    {%- endif %}
+{%- endblock %}
+
+{% block email_body -%}
```

Comment:
I think the whole block can be wrapped in a `dedent` filter in order to properly indent all the conditionals and make them readable ;)